src/tracer.js

```diff
@@ -267,7 +270,14 @@ export default class Tracer {
             throw new Error(`Unsupported format: ${format}`);
         }
 
-        injector.inject(spanContext, carrier);
+        if (spanContext && spanContext.context) {
```

Comment:
> That might be true, but opentracing compatibility passes null into this function, so we would have to change that first.

In this case we can do the check at the beginning of the method and throw exception, rather than keep checking throughout the method.